insonmnia/miner/server.go

```diff
@@ -123,7 +126,16 @@ func NewMiner(cfg *Config, opts ...Option) (m *Miner, err error) {
 	}
 
 	if o.dwh == nil {
-		o.dwh = dwh.NewDumbDWH(o.ctx)
+		_, TLSConfig, err := util.NewHitlessCertRotator(context.Background(), o.key)
```

Comment:
We already have initialized gRPC credentials into the `cmd/worker/main.go`. To prevent double work you can pass it via options.